codalab/server/worker_info_accessor.py

```diff
@@ -18,21 +19,32 @@ def wrapper(*args, **kwargs):
 
         return wrapper
 
-    def __init__(self, model, timeout_seconds):
+    def __init__(self, model, worker_model, timeout_seconds):
         self._model = model
+        self._worker_model = worker_model
         self._timeout_seconds = timeout_seconds
         self._last_fetch = None
         self._fetch_workers()
 
     def _fetch_workers(self):
-        self._workers = {worker['worker_id']: worker for worker in self._model.get_workers()}
+        self._workers = {worker['worker_id']: worker for worker in self._worker_model.get_workers()}
```

Comment:
What's the difference between `self._model.get_workers()` and `self._worker_model.get_workers()`?

Reply:
`_worker_model` has the logic to access to workers from the database, while `_model` is more general with access to other tables in the database. I needed both for this new feature.